platform/mellanox/docker-syncd-mlnx-rpc/Dockerfile.j2

```diff
@@ -48,7 +37,22 @@ RUN apt-get update \
     cmake               \
     libqt5core5a        \
     libqt5network5      \
-    libboost-atomic1.71.0
+    libboost-atomic1.74.0
+
+# Build and install python-scapy
+RUN curl http://ftp.us.debian.org/debian/pool/main/s/scapy/python-scapy_2.4.0-2_all.deb --output python-scapy_2.4.0-2_all.deb \
+ && dpkg -i python-scapy_2.4.0-2_all.deb \
+ && apt install -f
+
+{% if docker_syncd_mlnx_rpc_debs.strip() -%}
+# Copy locally-built Debian package dependencies
+{{ copy_files("debs/", docker_syncd_mlnx_rpc_debs.split(' '), "/debs/") }}
+{% endif %}
+
+{% if docker_syncd_mlnx_rpc_pydebs.strip() -%}
+# Copy locally-built Debian package dependencies
+{{ copy_files("python-debs/", docker_syncd_mlnx_rpc_pydebs.split(' '), "/debs/") }}
+{% endif %}
```

Comment:
Why moving this part?

Reply:
Need to install dependencies before installing these debs (due to python 2)

Reply:
I am talking about this part
```
{% if docker_syncd_mlnx_rpc_debs.strip() -%}
# Copy locally-built Debian package dependencies
{{ copy_files("debs/", docker_syncd_mlnx_rpc_debs.split(' '), "/debs/") }}
{% endif %}

{% if docker_syncd_mlnx_rpc_pydebs.strip() -%}
# Copy locally-built Debian package dependencies
{{ copy_files("python-debs/", docker_syncd_mlnx_rpc_pydebs.split(' '), "/debs/") }}
{% endif %}
```

It copied files, not installing.

Reply:
To have the copy adjacent to the installation. I made this change for better maintainability of this file.